src/parser/esprima_cpp/esprima.cpp

```diff
@@ -418,22 +418,37 @@ class Parser {
 
         VectorWithInlineStorage<8, SyntaxNode, std::allocator<SyntaxNode>>& params = paramsResult.params;
         bool hasParameterOtherThanIdentifier = false;
+        bool seenDefaultParameter = false;
 
         for (size_t i = 0; i < params.size(); i++) {
             switch (params[i]->type()) {
-            case Identifier: {
-                if (!hasParameterOtherThanIdentifier) {
+            case Identifier:
+                if (UNLIKELY(!seenDefaultParameter)) {
```

Comment:
It seems that `UNLIKELY` is used inappropriately here.
default parameter is not frequently used, so I think that `LIKELY` is more suitable for this case.

Reply:
You are right. I used a wrong macro. 
I have fixed it